src/Sulu/Bundle/WebsiteBundle/Analytics/AnalyticsManagerInterface.php

```diff
@@ -19,13 +19,13 @@
 interface AnalyticsManagerInterface
 {
     /**
-     * Returns all analytics for given webspace.
+     * Returns all analytics for given webspace and environment.
      *
-     * @param string $webspaceKey
+     * @param string $environment
      *
      * @return Analytics[]
      */
-    public function findAll($webspaceKey);
+    public function findAll($webspaceKey, $environment);
```

Comment:
would inject the `$environment` into the AnalyticsManager to avoid this BC Break in a Hotfix.